### PR TITLE
CLN: changed .format to fstring in pandas/io/formats/csvs.py

### PR DESCRIPTION
**Pull request by sardonick**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

https://github.com/pandas-dev/pandas/issues/29547

My first contribution.

### CODE REVIEW

File: pandas/io/formats/csvs.py [hunk 1]
```diff
@@ -237,12 +237,7 @@ def _save_header(self):
             return
         if has_aliases:
             if len(header) != len(cols):
-                raise ValueError(
-                    (
-                        "Writing {ncols} cols but got {nalias} "
-                        "aliases".format(ncols=len(cols), nalias=len(header))
-                    )
-                )
+                raise ValueError(f"Writing {len(cols)} cols but got {len(header)}")
```
Comment (MomIsBestFriend) [inline pandas/io/formats/csvs.py]: You forgot the word ```aliases``` at the end of the string.

**Comment by alimcmaster1** [comment]
Test failures related to @MomIsBestFriend comments above:

```
 with pytest.raises(ValueError, match=msg):
>               float_frame.to_csv(path, header=["AA", "X"])
E               AssertionError: Pattern 'Writing 4 cols but got 2 aliases' not found in 'Writing 4 cols but got 
```

**Comment by alimcmaster1** [comment]
Issue to ref here is https://github.com/pandas-dev/pandas/issues/29547

**Comment by jreback** [comment]
thanks @sardonick

**Pull request merged**